codegen/templates/workflow.tmpl

```diff
@@ -133,10 +134,24 @@ func (w {{$workflowStruct}}) Handle(
 	{{end}}
 
 	clientHeaders := map[string]string{}
+	{{if (ne (len $defaultHeaders) 0) }}
+	var ok bool
+	var h string
+	{{range $i, $k := $defaultHeaders}}
+	k := textproto.CanonicalMIMEHeaderKey("{{$k}}")
```

Comment:
You may not need to do this because `Get()` should handle it internally

Reply:
yeah, we need it either way, unfortunately.